x-pack/plugins/rollup/public/crud_app/sections/components/job_action_menu/job_action_menu.js

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiContextMenu,
+  EuiIcon,
+  EuiPopover,
+  EuiConfirmModal,
+  EuiOverlayMask
+} from '@elastic/eui';
+
+import { flattenPanelTree } from '../../../services';
+
+class JobActionMenuUi extends Component {
+  static propTypes = {
+    startJobs: PropTypes.func.isRequired,
+    stopJobs: PropTypes.func.isRequired,
+    deleteJobs: PropTypes.func.isRequired,
+    iconSide: PropTypes.string,
+    anchorPosition: PropTypes.string,
+    label: PropTypes.node,
+    iconType: PropTypes.string,
+    jobs: PropTypes.array,
+  }
+
+  static defaultProps = {
+    iconSide: 'right',
+    anchorPosition: 'rightUp',
+    iconType: 'arrowDown',
+    jobs: [],
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isPopoverOpen: false,
+      showDeleteConfirmation: false
+    };
+  }
+
+  panels() {
+    const {
+      startJobs,
+      stopJobs,
+      intl,
+    } = this.props;
+
+    const isSingleSelection = this.isSingleSelection();
+    const entity = this.getEntity(isSingleSelection);
+
+    const items = [];
+
+    if (this.canStartJobs()) {
+      items.push({
+        name: intl.formatMessage({
+          id: 'xpack.rollupJobs.jobActionMenu.startJobLabel',
+          defaultMessage: 'Start {entity}',
+        }, { entity }),
+        icon: <EuiIcon type="play" />,
+        onClick: () => {
+          this.closePopover();
+          startJobs();
+        },
+      });
+    }
+
+    if (this.canStopJobs()) {
+      items.push({
+        name: intl.formatMessage({
+          id: 'xpack.rollupJobs.jobActionMenu.stopJobLabel',
+          defaultMessage: 'Stop {entity}',
+        }, { entity }),
+        icon: <EuiIcon type="pause" />,
```

Comment:
I didn't understand that this was a pause symbol at first 😆I see that we have a `stop` icon as well, but not sure if that would be any more clear

Reply:
I think the stop icon makes sense since status is "stopped".

Reply:
![image](https://user-images.githubusercontent.com/1238659/44504851-146d3d80-a653-11e8-88a3-d60090d3a38f.png)